bites_test.py

```diff
@@ -596,5 +597,31 @@ def __init__(self, name, score):
     self.assertEqual(print_mock.call_args_list[1], mock.call("luigi: 9\n"))
     print_patcher.stop()
 
+class PlayFullGameTest(unittest.TestCase):
+  @patch('bites.Bites.take_all_turns')
+  def test_play_full_game_calls_take_all_turns(self, mock):
+  # test 82
+    bites_game = Bites([], {}, [])
+    bites_game.play_full_game()
+    self.assertTrue(mock.called)
+
+  @patch('bites.Bites.print_scores')
+  def test_play_full_game_calls_print_scores(self, mock):
```

Comment:
I suggest a more explicit variable name, i.e. "print_scores_mock" instead of "mock"